Casks/confluent-hub-client.rb

```diff
@@ -0,0 +1,23 @@
+cask 'confluent-hub-client' do
+  version '5.1.0-SNAPSHOT'
+  sha256 'a0c008f95cd3cd01b3a24729a27307ac5d01990cd9b73b185ccfb0728677af80'
+  url "https://s3-us-west-2.amazonaws.com/confluent-hub-client/ConfluentHubClient-#{version}.pkg"
```

Comment:
we shouldn't refer to this S3 url directly, we should have a URL that we can redirect as needed that just grabs from S3, same as we do for, e.g., docs.confluent.io. You can work with IT to get a subdomain setup if that is the best solution for now (or figure out how to get this hosted through an existing domain or section of our main website).

Reply:
Ok, will do so, thanks.